indico/modules/events/abstracts/templates/display/submit_abstract.html

```diff
@@ -0,0 +1,28 @@
+{% from 'events/display/indico/_common.html' import render_users %}
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_fieldset %}
+
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title -%}
+    {% trans %}Submit abstract{% endtrans %}
+{%- endblock %}
+
+{%- block content %}
+    {% if cfa.is_open %}
```

Comment:
imo it's better to just do 
```python
if not self.event_new.cfa.can_submit_abstracts(session.user):
    return WPSubmitAbstract.render_template('display/submit_abstract.html', self._conf, event=self.event_new,
                                            form=None)
```
at the top of the PR and just `{% if form %}` in the template..